libyul/AsmPrinter.cpp

```diff
@@ -256,30 +257,46 @@ string AsmPrinter::appendTypeName(YulString _type, bool _isBoolLiteral) const
 		return ":" + _type.str();
 }
 
-string AsmPrinter::formatSourceLocationComment(shared_ptr<DebugData const> const& _debugData,  bool _statement)
```

Comment:
Soon, this will have to handle all debug data, not only source locations. Should we undo this change in this PR or later?

Reply:
This was discussed on chat so marking as resolved. The overload that uses `DebugData` has not been removed, it's below.